python/lsst/ts/dimm/controllers/astelco_dimm.py

```diff
@@ -255,7 +259,14 @@ async def status_loop(self):
                 self.status["dec"] = dec.data[0]
                 self.status["altitude"] = altitude.data[0]
                 self.status["azimuth"] = azimuth.data[0]
-                self.log.debug(f"status: {self.status}")
+
+                # Deliberately logging the array so it is clear that a string
```

Comment:
Not sure I understand this comment. the f-string will deal with whatever type the input is. Consider removing it

Reply:
Done